### PR TITLE
Sdk l1 support docs

### PR DESCRIPTION
**Pull request by brikendr**
Updating documentation files to be in sync with the latest release of the sdk.

### CODE REVIEW

**Comment by brikendr** [comment]
For reviewers please check out the branch locally and run the following command:

```
npm install -g @compodoc/compodoc@latest
npm run compodoc
```
Then open your browser to the following link http://127.0.0.1:8080.

File: docs/index.html [hunk 1]
```diff
@@ -50,60 +49,82 @@ <h2 id="prerequisites">Prerequisites</h2>
...
+<p>With the node environment setup and ready, go ahead and install our sdk as part of your project dependencies:</p>
+<div><pre class="line-numbers"><code class="language-none">  npm install &#64;huddly/sdk&#64;latest</code></pre></div><p>You also need to install some other Huddly npm dependencies from which the SDK relies on for device discovery and communication. Depending on which product (camera) you will be working with, the corresponding device api packages must be accompanied with the SDK:</p>
+<div><pre class="line-numbers"><code class="language-none">  npm install &#64;huddly/device-api-usb    # For interacting with our USB cameras
+  npm install &#64;huddly/device-api-ip     # For interacting with our Ethernet/IP comeras</code></pre></div><p>It is also possible to install both of them and configure the SDK which one to use for discovery and/or communication.</p>
+<h3 id="setting-up-the-sdk-instance">Setting up the sdk instance</h3>
+<p>The example blow illustrates how to setup the SDK instance to communicate with a Huddly camera connected to the host machine. Start by creating the sdk and the transport apis:</p>
```
Comment (amfestoy) [inline docs/index.html:63]: below*:)

File: docs/interfaces/ICnnControl.html [hunk 2]
```diff
@@ -0,0 +1,511 @@
...
+            <code>src/interfaces/ICnnControl.ts</code>
+        </p>
+
+
+
+
+        <section>
+            <h3 id="index">Index</h3>
+            <table class="table table-sm table-bordered index-table">
+                <tbody>
```
Comment (amfestoy) [inline docs/interfaces/ICnnControl.html:79]: This is nothing that breaks, but just a comment if you wanted to learn. 
You have created a html table and grouped the body in <tbody>, if you want to take it to the next level I see that you have Methods that is a header. Would it maybe make sense to put this in a <theader>? If all the content is in the body the <tbody> tagg is not strictly needed:) this is not necessary, more like a semantic fun fact!

File: docs/interfaces/ICnnControl.html [hunk 3]
```diff
@@ -0,0 +1,511 @@
...
+is persistent on camera reboot/power cycle.</p>
+</div>
+
+                    <div class="io-description">
+                        <b>Parameters :</b>
+                        
+                        <table class="params">
+                            <thead>
+                                <tr>
+                                    <td>Name</td>
```
Comment (amfestoy) [inline docs/interfaces/ICnnControl.html:235]: Ah I see you used thead here so then I guess you allready knew how to use it:) To take this to the next level the children of a thead should be th and not td:)

File: docs/interfaces/ICnnControl.html (hunk 2 above)
Comment (brikendr) [reply docs/interfaces/ICnnControl.html:79]: @amfestoy this is generated code from compodoc. Please only review .ts, .md files.

File: README.md [hunk 4]
```diff
@@ -74,48 +99,50 @@ If you have a question or found a bug please [open an issue](https://github.com/
...
+#### **How can i disable autoframing and still get detection information from camera?**
   Another available detector configuration is the `shouldAutoFrame` option which when set to false, it allows you to configure Genius Framing to send detection data without autoframing.
 
   ```
     cameraManager.getDetector({ shouldAutoFrame: false });
   ```
   By default, the detector is configured to run autoframing and generate detection information.
 
-### How do I get detections only when I am streaming with my Huddly IQ!
+#### **How do I get detections only when I am streaming with my Huddly IQ!**
```
Comment (eskilgh) [inline README.md:130]: Huddly Camera?

File: README.md [hunk 5]
```diff
@@ -74,48 +99,50 @@ If you have a question or found a bug please [open an issue](https://github.com/
...
 
-### Transition from 0.3.* to 0.4.0
-  The transition from 0.3.* to 0.4.0 involves breaking changes with regards to Genius Framing configuration and detection data retrival. The breaking change is that we have moved the configuraion part of GeniusFraming (autozoom) into a separate interface that we called [AutozoomControl](http://developer.huddly.com/interfaces/IAutozoomControl.html#readme). The autozoom configuration methods `enable`, `disable`, `start` and `stop` which used to live in the detector interface are now moved to the new `AutozoomControl` interface. On the other hand, the [Detector](https://developer.huddly.com/interfaces/IDetector.html#readme) interace remains responsible for only setting up events for getting detection and framing data from the camera. Have a look at the documentation of each interface to see usage examples.
+#### **SDK crashes when there are two IQs connected!**
+The current implementation of the SDK does not support having two usb cameras connected to the host at the same time. However, having one Huddly usb cemera (IQ, Canvas, GO) and one Huddly IP camera (L1) is supported.
```
Comment (eskilgh) [inline README.md:148]: "having one Huddly usb **cemera** " typo

**Pull request merged**